index.bs

```diff
@@ -339,6 +351,16 @@ See <a href="https://lists.w3.org/Archives/Public/public-speech-api/2012Sep/0072
 
 </dl>
 
+<p>When the <dfn>start session algorithm</dfn> with <var>requestMicrophonePermission</var> is invoked, the user agent MUST run the following steps:
+
+1. If the [=current settings object=]'s [=relevant global object=]'s [=associated Document=] is NOT [=fully active=], throw an {{UnknownError}} and abort these steps.
```

Comment:
Note that I used `UnknownError` to [match WebKit](https://github.com/WebKit/WebKit/blob/1fb1a474fa4edeb7294e165044a3de7a3d977cf3/Source/WebCore/Modules/speech/SpeechRecognition.cpp#L78C43-L78C55) that already shipped Web Speech.

FYI Firefox uses `InvalidStateError` instead but did not ship Web Speech. So even though, I'd personally be in favor of `InvalidStateError`, I picked `UnknownError` so that web developers already catching this properly based on Safari don't have to update their code.

Reply:
I think webkit could align to whatever deemed appropriate, this is an edge case so this is likely not a web compat issue.
I also prefer `InvalidStateError`.

Reply:
Since you and @evanbliu also prefer `InvalidStateError`, I've updated this PR to use it.

Reply:
@youennf This PR has landed and web-platform-tests have been updated. Check out https://wpt.fyi/results/speech-api/SpeechRecognition-detached-iframe.window.html?label=experimental&label=master&aligned
It would be nice to update Safari's implementation as well.